samples/console_output/main.cpp

```diff
@@ -154,7 +159,9 @@ class TimeChecker : public sensr::MessageListener {
   void OnError(Error error, const std::string& reason) override {
     (void)reason;
```

Comment:
Do we ever use the `reason`? Every `OnError` seems to just cast it to void

Reply:
Let me change it to print the error reason.
`cout << reason << endl;`

Reply:
Wouldn't `cerr << reason << endl` be more appropriate?